test/transcoding/FPGAIVDepLoopAttrOnClosure.ll

```diff
@@ -0,0 +1,553 @@
+; This LLVM IR was generated using Intel SYCL Clang compiler (https://github.com/intel/llvm)
+;
+; SYCL source code of this program can be found below.
+;
+; template<typename N, typename Func>
+; __attribute__((sycl_kernel))
+; void kernel(Func f) {
+;   f();
+; }
+;
+; int main() {
+;   int buf1[10], buf2[10];
+;   const int c = 42;
+;
+;   kernel<class EmbeddedLoopTest>([=]() mutable {
+;     [[intelfpga::ivdep(buf1, 3)]]
+;     for (int i = 0; i < 6; ++i) {
+;       buf1[i] *= (buf2[i + 4] + c);
+;       [[intelfpga::ivdep(2)]]
+;       for (int j = 0; j < 7; ++j)
+;         buf2[i] *= (buf1[i] + buf2[i + 3]);
+;     }
+;   });
+;
+;   kernel<class VaryingSafelenTest>([=]() mutable {
+;     [[intelfpga::ivdep(buf1, 3)]]
+;     [[intelfpga::ivdep(buf2, 2)]]
+;     for (int i = 0; i < 6; ++i) {
+;       buf1[i] *= (buf2[i + 4] + c);
+;       buf2[i] *= (buf1[i] + buf2[i + 3]);
+;     }
+;   });
+;
+;   return 0;
+; }
+
+; RUN: llvm-as < %s > %t.bc
+; RUN: llvm-spirv %t.bc --spirv-ext=+SPV_INTEL_fpga_loop_controls -o %t.spv
+; RUN: llvm-spirv -to-text %t.spv -o %t.spt
+; RUN: FileCheck < %t.spt %s --check-prefix=CHECK-SPIRV
```

Comment:
CHECK-SPIRV does almost nothing in the test.

Reply:
Ah, wait, there is a TODO above

Reply:
Yep, coming soon!